tidb/upgrade.bash.ticat

```diff
@@ -0,0 +1,24 @@
+help = upgrade a specified TiDB cluter
+abbr = update
+
+[arg]
+cluster-name|name|n|N = ''
+version|v|V = ''
+confirm|cfm = ''
+force = false
+ignore-config-check|ignore-check|skip-check = false
+offline|off = false
+
+[arg2env]
+tidb.cluster = cluster-name
+tidb.version = version
+tidb.op.confirm = confirm
```

Comment:
In a second thought, maybe include all args in `arg2env` is a better practice:
That means adding these to `arg2env`:
```
force
ignore-config-check
offline
```